hack/grpc-protoc.Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is used to build an image with the Protobuf compiler and the Go Protobuf plugin together. This image
+# is not published from this repository, but is used to generate our protobuf Go code. This is nessicary as we need to
+# use spesific pinned versions of both protoc and the Go plugin to generate the correct output.
```

Comment:
```suggestion
# use spesific pinned versions of both Protobuf and the Go plugin to generate the correct output.
```

Reply:
I think it's accurate to say `protoc` here. We're talking about the compiler in particular.